rasa/shared/core/domain.py

```diff
@@ -564,7 +567,7 @@ def retrieval_intents(self) -> List[Text]:
     @classmethod
     def collect_entity_properties(
         cls, domain_entities: List[Union[Text, Dict[Text, Any]]]
-    ) -> Tuple[List[Text], Dict[Text, List[Text]], Dict[Text, List[Text]]]:
+    ) -> Tuple[List[Text], Dict[Text, List[Text]], Dict[Text, List[Text]], List[Text]]:
```

Comment:
The docstring is missing info on the 4th returned element (the list of default ignored entities).

Reply:
Resolved as part of making it a dataclass